connect/runtime/src/test/java/org/apache/kafka/connect/integration/StartAndStopCounter.java

```diff
@@ -72,6 +72,10 @@ public int stops() {
         return stopCounter.get();
     }
 
+    public StartAndStopCounterSnapshot countsSnapshot() {
```

Comment:
I found the name a bit overloaded and added a suggestion below. You think we could make it a bit simpler?